app/views/helpers/_post.html.erb

```diff
@@ -0,0 +1,17 @@
+<li>
+  <div class='user-post'>
+    <div class='user-details'>
+      <h3><%= post.title %></h3>
+      <p><%= post.text %></p>
+    </div>
+    <div class='comments-likes'>
+      <span>
+        Comments:<%= post.comments_counter.nil? ? '0' : post.comments_counter %>
+      </span>
+      <span>
+        Likes:<%= post.likes_counter.nil? ? '0' : post.likes_counter %>
+      </span>
+    </div>   
+  </div>
+  <%= render 'helpers/comments', post: post %>
```

Comment:
- [ ] Great job using form helper for all your forms. It's really a good practice and I give you kudos for doing so. You're great man. However, there's a little mistake here, I think this should be `helpers/all_comments` not `helpers/comments`. I got this error with your present reference, but worked fine when I changed it to `helpers/all_comments`. kindly fix it.

[comment-ref.webm](https://github.com/2004-okari/BLOG_APP/assets/26597432/9d256e65-0bc8-44a0-b050-934262b00b1b)

Reply:
Thanks for the feedback👍👍. The issue was also bugging me